DOCS-12189-mmp-shared-lib-v4.4-cv (#2123)
mulesoft/docs-mule-runtime-3: modules/ROOT/pages/mmp-concept.adoc

[[configure-shared-libraries]]
== Configure Shared Libraries

Shared libraries enable you to configure a dependency in your application and make this dependency accessible to all the connectors in your application. +

> See xref:mule-runtime::about-classloading-isolation.adoc#shared-libraries[Shared Libraries] for an in-depth explanation about this feature.

Reviewer: "explanation of" is more common than "explanation about"

> To configure a dependency as a shared library, add the following elements in your Mule Maven plugin configuration inside your project's `pom.xml` file:

Reviewer: I got a little confused here by "on" and "inside"
Are we asking the audience to add the elements **to** the plugin config or to the POM?

[[configure-plugin-dependencies]]
== Configure Additional Plugin Dependencies

> You can configure additional plugin dependencies for a connector in your application, which makes the dependency available only for that connector. This configuration enables you to configure different dependencies, or even multiple versions of the same dependency, for different connectors. +

Reviewer: Be sure to standardize to either "dependency" or "dependencies" in the first sentence.
Just an alternative:
```suggestion
Configuring additional plugin dependencies for a connector in your application makes those dependencies available for only that connector. Doing this enables you to configure different dependencies, or even multiple versions of the same dependency, for different connectors. +
```

> See xref:mule-runtime::about-classloading-isolation.adoc#additional-plugin-dependencies[Additional Plugin Dependencies] for an in-depth explanation about this feature.

Reviewer: ```suggestion
See xref:mule-runtime::about-classloading-isolation.adoc#additional-plugin-dependencies[Additional Plugin Dependencies] for an in-depth explanation of this feature.
```
